client/components/support-user/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Or maybe better have a `client/support` folder where we can also put `client/me/help` if it makes sense. The goal being being that `components` is more suitable for UI components not connected to out state tree.